docs/site/home.html

```diff
@@ -95,6 +104,14 @@ <h3 class="block-title">What is it?</h3>
   console.log(err, entity);
 });</div>
 
+        <div hljs language="js" ng-show="selected == 'compute engine'">
+// Auth is handled automatically inside GCE.
+var dataset = require('gcloud').datastore.dataset();
```

Comment:
Our code samples should always reflect our best practices too. I believe creating a separate `gcloud` object is always what we should suggest to the users. So the code should be fixed as such:

``` js
var gcloud = require('gcloud');
var dataset = gcloud.datastore.dataset();
// ... rest as normal
```